commit 631ce4c5e71808e0922259947a31e59f73d836e0 (refs/changes/98/798/2)
Author: Trent Mick <trent.mick@joyent.com>
Date:   2016-10-27T15:58:39-07:00 (2 years, 11 months ago)
    
    joyent/node-manta#277 mjob fails with mjob: AssertionError: body (object) is required
    Reviewed by: Angela Fong <angela.fong@joyent.com>
    Approved by: Angela Fong <angela.fong@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 4053dce..a1a936b 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,7 +2,14 @@
 
 ## not yet released
 
-- joyent/node-manta#270 Add -p to man mput
+## 3.1.3
+
+- joyent/node-manta#277 mjob fails with "mjob: AssertionError: body (object) is required"
+
+## 3.1.2
+
+- joyent/node-manta#275 msign with subusers broken
+- joyent/node-manta#270 Add -p to `man mput`
 
 ## 3.1.1
 
@@ -19,7 +26,8 @@
 
 ## 3.0.0
 
-- joyent/node-manta#246 Update many dependencies to come up to the modern age.
+- joyent/node-manta#246 Update many dependencies to support node v4 and v6
+  without build errors or warnings.
   This involved dropping support for node 0.8. It is for this reason, and
   prudence at the large number of dependency updates (many of them across
   major version bumps) that we are doing a major version bump of this package.
diff --git a/package.json b/package.json
index 5af519b..85df63c 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
         "type": "git",
         "url": "git://github.com/joyent/node-manta.git"
     },
-    "version": "3.1.2",
+    "version": "3.1.3",
     "main": "./lib/index.js",
     "dependencies": {
         "assert-plus": "^1.0.0",
@@ -27,7 +27,7 @@
         "path-platform": "~0.0.1",
         "progbar": "^1.1.0",
         "readable-stream": "~1.1.9",
-        "restify-clients": "~1.3.0",
+        "restify-clients": "~1.4.1",
         "showdown": "~1.4.2",
         "smartdc-auth": "^2.4.1",
         "tar": "~2.2.1",
